### DISCUSSION TITLE
What is the proper way to start a FDS parallel run on Linux clusters?

### DISCUSSION THREAD

**Discussion by Yunlongjasonliu** [firemodels/fds#13619] [General]
Why I use the following my parallel case won't run?

mpiexec -n 5 job_name.fds

But using the following it works, is the following the correct way to start a run?

mpiexec -n 5 fds job_name.fds

**Comment by drjfloyd** [comment]
mpiexec - n 5 job_name.fds  
This command tells your computer to run the executable named job_name.fds with 5 processes.  job_name.fds, however, is not an executable it is your input file.

Please read the FDS documentation. There are instructions on how to run FDS in it.

**Comment by Yunlongjasonliu** [comment]
It only works on my Linux platform if I applied the following:
mpiexec -n 5 fds job_name.fds

**Reply by drjfloyd** [reply]
That is how it is supposed to work.  
mpiexec -n 5 fds job_name.fds
That input tells the machine to run the program fds with 5 mpi processes and pass in the string job_name.fds as a command line parameter
